docs/releases/v0.31.0.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+## [v0.31.0](https://github.com/aws-observability/aws-otel-collector/tree/v0.31.0) (2023-07-11)
+
+[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.30.0...v0.31.0)
+
+**Merged pull requests:**
+
+- Remove prometheus normalization feature gate [\#2178](https://github.com/aws-observability/aws-otel-collector/pull/2178) ([bryan-aguilar](https://github.com/bryan-aguilar))
```

Comment:
The gate hasn't been removed, has it?  Just the forcible resetting of its status to alpha since the change to beta was reverted upstream?

Reply:
Good feedback, let me reword this.

Reply:
Updated